GUI/Views/DownloadUniProtDatabaseWindow.xaml.cs

```diff
@@ -55,9 +56,11 @@ private void DownloadProteomeWithProgress(string DownloadLink, string TargetPath
             {
                 WindowStartupLocation = WindowStartupLocation.CenterScreen
             };
-            progress.Show();
+            progress.Show();    
             Task.Factory.StartNew(() =>
             {
+                client.Timeout = TimeSpan.FromMinutes(30);
+                AppContext.SetSwitch("System.Net.Http.SocketsHttpHandler.Http2UnencryptedSupport", true);
                 HttpResponseMessage urlResponse = Task.Run(() => client.GetAsync(uri)).Result;
```

Comment:
I know you didn't write this, but it needs to be changed. If you use an Async method, it should include a call to either Wait or Await

Reply:
https://blog.stephencleary.com/2012/07/dont-block-on-async-code.html

https://blog.stephencleary.com/2012/02/async-and-await.html

Based on those, I think we should use await so we don't lock up the UI thread.